developers_chamber/bitbucket_utils.py

```diff
@@ -5,14 +5,39 @@
 from click import UsageError
 
 
-def create_merge_release_pull_request(username, password, source_branch_name, destination_branch_name, repository_name):
-    url = 'https://api.bitbucket.org/2.0/repositories/{repository}/pullrequests'.format(
+def get_current_user_uuid(username, password):
```

Comment:
tohle je celé na hlavu, když vytváříme pr přes api tak to nepřidá default reviewers, musí se stáhnout a přidat ručně do requestu. Nicméně tam nemůže být zas aktuální uživatel takže nejprve musím stáhnout jeho uuid abych jej mohl vyloučit